### ISSUE TITLE
'NoneType' object has no attribute 'split'

### ISSUE THREAD

**Issue by maxime-pigeon** [sublimelsp/LSP-TexLab#8]
I get this error after installing the package and trying to open a TeX file.

## Version
 - LSP: 1.10.0
 - Sublime Text: 4113
## Server Test Run
 - exit code: -1
 - output
```
'NoneType' object has no attribute 'split'
```
## Server Configuration
 - command
```json
[
  "/Users/maximepigeon/Library/Caches/Sublime Text/Package Storage/LSP-TexLab/osx-arm64~v3.2.0/texlab"
]
```
 - shell command
```sh

```
 - selector
```
text.tex | text.bibtex
```
 - priority_selector
```
text.tex | text.bibtex
```
 - init_options
```json
{}
```
 - settings
```json
{
  "texlab": {
    "auxDirectory": ".", 
    "bibtexFormatter": "texlab", 
    "build": {
      "args": [
        "-pdf", 
        "-interaction=nonstopmode", 
        "-synctex=1", 
        "%f"
      ], 
      "executable": "latexmk", 
      "forwardSearchAfter": false, 
      "isContinuous": false, 
      "onSave": false
    }, 
    "chktex": {
      "onEdit": false, 
      "onOpenAndSave": false
    }, 
    "diagnosticsDelay": 300, 
    "formatterLineLength": 80, 
    "forwardSearch": {
      "args": [], 
      "executable": null
    }, 
    "latexFormatter": "latexindent", 
    "latexindent": {
      "local": null, 
      "modifyLineBreaks": false
    }, 
    "rootDirectory": null
  }
}
```
 - env
```json
{}
```

## Active view
 - File name
```
/Users/maximepigeon/Desktop/Thesis/Chapter 00/Section 01.tex
```
 - Settings
```json
{
  "auto_complete_selector": "meta.tag, source - comment - string.quoted.double.block - string.quoted.single.block - string.unquoted.heredoc", 
  "lsp_active": null, 
  "syntax": "Packages/LaTeX/LaTeX.sublime-syntax"
}
```
 - base scope
```
text.tex.latex
```

## Project / Workspace
 - folders
```json
[
  "/Users/maximepigeon/Desktop/Thesis"
]
```
 - is project: False

## LSP configuration

<not found>
## System PATH
 - /opt/homebrew/bin
 - /opt/homebrew/sbin
 - /usr/local/bin
 - /usr/bin
 - /bin
 - /usr/sbin
 - /sbin
 - /Library/TeX/texbin

**Comment by jfcherng** [comment]
There is no arm64 build. https://github.com/latex-lsp/texlab/releases

**Comment by maxime-pigeon** [comment]
@jfcherng Thanks for the quick response! I am confused too. I use TexLab globally with BBEdit just fine (installed the server through Homebrew: https://formulae.brew.sh/formula/texlab). 🤔

**Comment by jfcherng** [comment]
> installed the server through Homebrew: [formulae.brew.sh/formula/texlab](https://formulae.brew.sh/formula/texlab)

What's the output of `which texlab` in an external terminal?

**Comment by maxime-pigeon** [comment]
/opt/homebrew/bin/texlab
('4113', 'osx_arm64', '7be878dce68c856b4bf3045e18f08015')

**Comment by jfcherng** [comment]
Could you test [this branch](https://github.com/sublimelsp/LSP-TexLab/tree/arm64-test) with LSP-texlab.sublime-settings:

```js
{
	"command": "texlab",
}
```

or

```js
{
	"command": "/opt/homebrew/bin/texlab",
}
```

**Comment by maxime-pigeon** [comment]
> Still don't know why osx_arm64 can install this package via Package Control though.

Ah, I couldn't install directly through Package Control; I had to add the repository and then install it. Maybe that explains it. Sorry, I should have mentioned it outright.

I cloned the test branch in my packages directory but I can't enable the server in my project (it doesn't appear in the list). Maybe I'm doing something wrong?

**Comment by jfcherng** [comment]
> I cloned the test branch in my packages directory but I can't enable the server in my project (it doesn't appear in the list). Maybe I'm doing something wrong?

It should auto work unless you disable it intentionally.

What's the path of `plugin.py`?

**Comment by jfcherng** [comment]
Should've been fixed in 0.5.1
